modules/pytket-honeywell/tests/backend_test.py

```diff
@@ -374,3 +374,32 @@ def test_submission_with_group() -> None:
     shots = b.run_circuit(c, n_shots=n_shots, group="test").get_shots()  # type: ignore
     print(shots)
     assert all(q[0] == q[1] for q in shots)
+
+
+def test_zzphase() -> None:
```

Comment:
Should this test be run every time in the ci?
The backend looks like the real device to me?
Maybe add  `@pytest.mark.skipif(skip_remote_tests, reason=REASON)`

Reply:
good spot, thanks